api/server/volume.go

```diff
@@ -1811,11 +1815,19 @@ func (vd *volAPI) SetupRoutesWithAuth(
 	routes = append(routes, vd.otherVolumeRoutes()...)
 	routes = append(routes, vd.snapRoutes()...)
 	routes = append(routes, vd.backupRoutes()...)
```

Comment:
Should we also secure these routes?

Reply:
No they are not intended to be secure. We need to secure non-volume calls.